extensions/wikia/UserActivity/js/UserActivity.js

```diff
@@ -0,0 +1,73 @@
+require([
+	'jquery'
+], function ($) {
+
+	'use strict';
+
+	$(function () {
+		var orderMatch = /order=([^:]+):(desc|asc)/i,
+			results,
+			curOrder,
+			curDirection,
+			curLocation = window.location;
+
+		results = curLocation.href.match(orderMatch);
+		if (results) {
+			curOrder = results[1] || 'lastedit';
+			curDirection = results[2] || 'desc';
+		}
+
+		$('#title-header')
+			.addClass(function () {
+				return updateHighlightFor('title');
+			})
+			.click(function () {
+				updateOrderFor('title');
+			});
+		$('#edits-header')
+			.addClass(function () {
+				return updateHighlightFor('edits');
+			})
+			.click(function () {
+			updateOrderFor('edits');
+		});
+		$('#last-edit-header')
+			.addClass(function () {
+				return updateHighlightFor('lastedit');
+			})
+			.click(function () {
+			updateOrderFor('lastedit');
+		});
+
+		function updateHighlightFor(column) {
+			if (typeof curOrder === 'undefined' && column === 'lastedit') {
```

Comment:
Maybe we should initialize ```curOrder``` and ```column``` with defaults to avoid this special condition here?